### PR TITLE
[clad] Bump clad version to v1.3.

### PR DESCRIPTION
**Pull request by vgvassilev**
This new release includes performance and stability improvements.

See more at: https://github.com/vgvassilev/clad/releases/tag/v1.3

### CODE REVIEW

**Comment by vgvassilev** [comment]
@guitargeek, can you remind me which was the test case that we had disabled on Windows and we found out that it mostly works now? I thought it was `testRooFuncWrapper.cxx` but I could not find any windows-related ifdefs...

File: roofit/histfactory/test/testHistFactory.cxx [hunk 1]
```diff
@@ -774,6 +774,7 @@ INSTANTIATE_TEST_SUITE_P(HistFactory, HFFixtureFit,
                                           testing::Values(ROOFIT_EVAL_BACKENDS)),
                          getNameFromInfo);
 
+#ifndef R__WIN32 // See https://github.com/vgvassilev/clad/issues/752
```
Comment (guitargeek) [inline roofit/histfactory/test/testHistFactory.cxx:777]:
```suggestion
#if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS) // See https://github.com/vgvassilev/clad/issues/752
```
The preprocessor guard to exclude test from Windows is usually that one

**Comment by guitargeek** [comment]
> @guitargeek, can you remind me which was the test case that we had disabled on Windows and we found out that it mostly works now? I thought it was `testRooFuncWrapper.cxx` but I could not find any windows-related ifdefs...

You don't see any ifdefs because the whole file is excluded at the CMake level:
https://github.com/root-project/root/blob/master/roofit/roofitcore/test/CMakeLists.txt#L51

However, in the HistFactory tests, your are doing the right thing by only excluding the AD tests :+1:

**Review by guitargeek — APPROVED** [review]
LGTM, thanks a lot for the update and following up so quickly on the review!

**Comment by vgvassilev** [comment]
I wanted to test out re-enabling this other test and only suppressing whichever part of it fails...

**Comment by vgvassilev** [comment]
> Build failed on windows10/default. Running on null:C:\build\workspace\root-pullrequests-build [See console output](https://lcgapp-services.cern.ch/root-jenkins/job/root-pullrequests-build/195088/console).
> 
> ### Failing tests:
> * [projectroot.roofit.roofitcore.test.gtest_roofit_roofitcore_test_testRooFuncWrapper](https://lcgapp-services.cern.ch/root-jenkins/job/root-pullrequests-build/195088/testReport/projectroot.roofit.roofitcore/test/gtest_roofit_roofitcore_test_testRooFuncWrapper/)

@guitargeek, could you take a look at the failure and maybe suggest what needs suppressing on windows?

**Comment by guitargeek** [comment]
It's at the end of the `testRooFuncWrapper` file where you can exclude specific tests:
https://github.com/root-project/root/blob/master/roofit/roofitcore/test/testRooFuncWrapper.cxx

In the case of Windows, it's the `AddPdf` fit that fails, which is encoded in `param3`. So wee need:
```c++
INSTANTIATE_TEST_SUITE_P(RooFuncWrapper, FactoryTest,
                         testing::Values(param1, param2,
#if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS)
                                         param3,
#endif
                                         param4, param5, param6, param7, param8, param8p1, param9, param10, param11,
                                         param12, param13),
                         [](testing::TestParamInfo<FactoryTest::ParamType> const &paramInfo) {
                            return paramInfo.param._name;
                         });
```

By the way, this failure is maybe not even related to Clad, but could be because of different random numbers. I will follow up on this later after this PR.

**Comment by vgvassilev** [comment]
> It's at the end of the `testRooFuncWrapper` file where you can exclude specific tests: https://github.com/root-project/root/blob/master/roofit/roofitcore/test/testRooFuncWrapper.cxx
> 
> In the case of Windows, it's the `AddPdf` fit that fails, which is encoded in `param3`. So wee need:
> 
> ```c++
> INSTANTIATE_TEST_SUITE_P(RooFuncWrapper, FactoryTest,
>                          testing::Values(param1, param2,
> #if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS)
>                                          param3,
> #endif
>                                          param4, param5, param6, param7, param8, param8p1, param9, param10, param11,
>                                          param12, param13),
>                          [](testing::TestParamInfo<FactoryTest::ParamType> const &paramInfo) {
>                             return paramInfo.param._name;
>                          });
> ```
> 
> By the way, this failure is maybe not even related to Clad, but could be because of different random numbers. I will follow up on this later after this PR.

I do not understand the change but I've added it. Let's see what happens.

**Comment by vgvassilev** [comment]
I do not see how the GitHub actions failures are related to this PR. Let’s move forward, jenkins is green.

**Pull request merged**